lib/Widgets/Utils.vala

```diff
@@ -169,11 +169,16 @@ public static string accel_to_string (string? accel) {
             arr += _("Right Shift");
             break;
         default:
-            arr += Gtk.accelerator_get_label (accel_key, 0);
+            // If a specified accelarator contains only modifiers e.g. "<Control><Shift>",
+            // we get anything from accelerator_get_label method, so skip that case
```

Comment:
Do you mean "we don't get anything ..."?

Reply:
Yeah you're right. Sorry for my poor English 😅